src/adapter.ts

```diff
@@ -37,49 +34,44 @@ export class SequelizeAdapter implements Adapter {
      * If dbSpecified == true, you need to make sure the DB in connStr exists.
      * If dbSpecified == false, the adapter will automatically create a DB named 'casbin'.
      */
-    public static async newAdapter(connStr: string, dbSpecified: boolean = false) {
-        const a = new SequelizeAdapter(connStr, dbSpecified);
+    public static async newAdapter(param: string | Sequelize, dbSpecified: boolean = false) {
```

Comment:
I know this code works, but it is not very consistent with how we did it for other adapters like the Xorm Adapter for Go: https://github.com/casbin/xorm-adapter/blob/master/adapter.go#L77-L101

I suggest we still use the original flow.